.github/ISSUE_TEMPLATE/a_question.yml

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Question 🤔
+description: Usage question or discussion about Fabric8 kubernetes-client
+labels: [kind/question]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Asking a new Question 🤔
+        🔍  Before opening a new issue please search existing issues at https://github.com/fabric8io/kubernetes-client/issues
+
+        🤔  To make it easier for us to help you, please include as much useful information as possible.
+
+        Useful Links:
+
+          - 📄  Documentation: https://fabric8.io/
```

Comment:
Shall we replace this with CHEATSHEET URL?

https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md

Reply:
sure.

Reply:
Modified.